### PR TITLE
Feat/issue 33 add items to orderpage

### PR DESCRIPTION
**Pull request by hellodebs**
Hello,
you can now add the items from the Menu.js and display them in Order.js. There are still tiny things I need to fix but this works.

### CODE REVIEW

File: src/components/ItemImage.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.item__image {
```
Comment (adrianammt) [inline src/components/ItemImage.css:1]: Maybe you can find another way to scale or fill the background with the pictures, some pictures look a bit off.

File: src/pages/Menu.js [hunk 2]
```diff
@@ -1,8 +1,14 @@
...
 
-export default function Menu({ items }) {
+export default function Menu({ items, updateItemQuantity }) {
   return items.map((item) => {
-    return <MenuItem key={item.id} item={item} />;
+    return (
+      <MenuItem
+        key={item.id}
+        item={item}
+        updateItemQuantity={updateItemQuantity}
```
Comment (adrianammt) [inline src/pages/Menu.js:10]: Another nice feature for the future could be that the chosen quantity stays fixed, so you can edit and then it sets to 0 once you confirm your order. Right now it sets to 0 when I go to the shopping cart, so if say I want to increase my order of Bungo Bbang I have to go back and forth to see how many I had and then increase. Just a suggestion.

File: src/hooks/useCount.js [hunk 3]
```diff
@@ -1,15 +1,17 @@
 import { useState } from "react";
 
-export default function useCount() {
+export default function useCount(itemId, updateItemQuantity) {
   const [count, setCount] = useState(0);
 
   function increment() {
     setCount(count + 1);
+    updateItemQuantity(itemId, count + 1);
```
Comment (adrianammt) [inline src/hooks/useCount.js:8]: I think it would be a good idea to have a max amount of items I can order. I just ordered 202 dishes of Mandu. :)

**Review by ScienceMaster7 — APPROVED** [review]
I like how well structured your code is.

Well done !

**Review by adrianammt — COMMENTED** [review]
Good job Debs! Left you a couple of suggestions. ☀️

File: src/components/ItemImage.css (hunk 1 above)
Comment (hellodebs) [reply src/components/ItemImage.css:1]: I will replace some pics. :) I need to find good ones.

File: src/pages/Menu.js (hunk 2 above)
Comment (hellodebs) [reply src/pages/Menu.js:10]: that's exactly what I still need to fix :D Don't know how yet.

File: src/hooks/useCount.js (hunk 3 above)
Comment (hellodebs) [reply src/hooks/useCount.js:8]: hahahaahaha!!! :D okay got it

**Pull request merged**